### PR TITLE
[ADAM-1806] Add toVariants conversion from GenotypeRDD.

### LINKED ISSUE #1806: GenotypeRDD should support `toVariants` method

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #1806.

### CODE REVIEW

**Comment by fnothaft** [comment]
Just realized, this needs to propagate through to R/Python as well.

**Comment by heuermh** [comment]
> Just realized, this needs to propagate through to R/Python as well.

Yep, at least we haven't spread ourselves too thin.  ;)

**Comment by fnothaft** [comment]
This is good to go from my side now.

File: adam-python/bdgenomics/adam/rdd.py [hunk 1]
```diff
@@ -874,6 +874,22 @@ def toVariantContexts(self):
         return VariantContextRDD(vcs, self.sc)
 
 
+    def toVariants(self, dedupe=False):
+        """
+        Extracts the variants contained in this RDD of genotypes.
+
+        Does not perform any filtering looking at whether the variant was called
+        or not. By default, does not deduplicate variants. By default, does not
+        deduplicate variants.
```
Comment (heuermh) [inline adam-python/bdgenomics/adam/rdd.py]: Doc comment needs deduplicating.

Comment (fnothaft) [reply adam-python/bdgenomics/adam/rdd.py]: Ironic!

**Comment by fnothaft** [comment]
Addressed reviewer comments.

**Comment by heuermh** [comment]
Thank you, @fnothaft

**Pull request merged**